[BUG] TypeScript error "Cannot find name 'BaseTableVirtuosoHandle'"

<!-- I don't provide free support through Github issues. If you want to ask for support or request features, [sponsor the project](https://github.com/sponsors/petyosi) and contact me over email.
-->

**Describe the bug**

When type-checking a react-virtuoso project with `skipLibCheck` set to `false` it fails with the following errors:

```
./node_modules/react-virtuoso/dist/index.d.ts:429:61 - error TS2552: Cannot find name 'BaseTableVirtuosoHandle'. Did you mean 'TableVirtuosoHandle'?

429 export declare interface GroupedTableVirtuosoHandle extends BaseTableVirtuosoHandle {
                                                                ~~~~~~~~~~~~~~~~~~~~~~~

./node_modules/react-virtuoso/dist/index.d.ts:1089:54 - error TS2304: Cannot find name 'BaseTableVirtuosoHandle'.

1089 export declare interface TableVirtuosoHandle extends BaseTableVirtuosoHandle {
```

**Reproduction**

CodeSandbox seems very buggy at the moment, I keep getting CodeSandbox errors. I can create a GitHub repo if desired.

**To Reproduce**

Steps to reproduce the behavior:
1. Create a TS project that uses `react-virtuoso@4.18.0`
2. Run `tsc --skipLibCheck false`
4. See errors

**Expected behavior**

No type-checking errors

**Screenshots**

N/A

**Desktop (please complete the following information):**

- OS: N/A
- Browser: N/A

**Additional context**
None

--- a/.changeset/spicy-numbers-feel.md
+++ b/.changeset/spicy-numbers-feel.md
@@ -1,0 +1,5 @@
+---
+"react-virtuoso": patch
+---
+
+Fix missing type for table

--- a/packages/react-virtuoso/src/component-interfaces/TableVirtuoso.ts
+++ b/packages/react-virtuoso/src/component-interfaces/TableVirtuoso.ts
@@ -20,9 +20,6 @@ import type {
 } from '../interfaces'
 import type { VirtuosoProps } from './Virtuoso'
 
-/**
- * @internal
- */
 interface BaseTableVirtuosoHandle {
   /**
    * Obtains the internal size state of the component, so that it can be restored later. This does not include the data items.

